src/index.android.js

```diff
@@ -0,0 +1,22 @@
+import { NativeModules } from "react-native";
+
+const { NavigationBarColor } = NativeModules;
+
+const changeNavigationBarColor = (
+  color = String,
+  light = false,
+  animated = true
+) => {
+  const LightNav = light ? true : false;
+  NavigationBarColor.changeNavigationBarColor(color, LightNav, animated);
```

Comment:
This const is useless.
```suggestion
  NavigationBarColor.changeNavigationBarColor(color, light, animated);
```

Reply:
You can fix #36 here if return changeNavigationBarColor results like in https://github.com/thebylito/react-native-navigation-bar-color/pull/37/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R12